client/setup.go

```
@@ -11,23 +13,19 @@ import (
 )
 
 type Client struct {
-	TweetBot   *twitter.Client
-	DiscordBot *discordgo.Session
-	ShutDown   chan os.Signal
+	TweetBot       *twitter.Client
+	TwitterClients map[string]*twitter.Client
+	DiscordBot     *discordgo.Session
+	ShutDown       chan os.Signal
 }
 
 func NewClient() (*Client, error) {
-	oauthConsumerKey := os.Getenv("OAUTH_CONSUMER_KEY")
-	oauthConsumerSecret := os.Getenv("OAUTH_CONSUMER_SECRET")
-	oauthAccessToken := os.Getenv("OAUTH_ACCESS_TOKEN")
-	oauthAccessSecret := os.Getenv("OAUTH_ACCESS_SECRET")
-	config := oauth1.NewConfig(oauthConsumerKey, oauthConsumerSecret)
-	token := oauth1.NewToken(oauthAccessToken, oauthAccessSecret)
 	discordToken := os.Getenv("DISCORD_TOKEN")
-	httpClient := config.Client(oauth1.NoContext, token)
 
-	// Twitter client
-	client := twitter.NewClient(httpClient)
+	tClients, err := SetupTwitterClients(os.Getenv("CREDENTIAL_FILE"))
+	if err != nil {
+		log.Println("credential file support not implemented")
```

print the error message out too?

---

Return the error, instead of logging?